app/views/users/_form.html.erb

```diff
@@ -88,7 +88,8 @@
     <% end %>
 
     <div class='tab-pane' id='roles'>
-      <%= checkbox_f f, :admin if User.current.can_change_admin_flag? %>
+      <% caption = @user.inherited_admin? ? "#{_('Admin rights are currently inherited from')} #{_('Usergroup')}" : '' %>
```

Comment:
This will need a fix, the translated string without interpolation. Usergroup should be translated duplicitly in this string, some languages don't keep subject at the end. It should be

_('Admin rights are currently inherited from a user group')